contracts/UsingTellor.sol

```diff
@@ -260,54 +273,7 @@ contract UsingTellor is IERC2362 {
             uint256 _statusCode
         )
     {
-        bool _isERC362Id;
-        uint256 _decimalsDividend = 1;
-        if (
-            _id ==
-            0xdfaa6f747f0f012e8f2069d6ecacff25f5cdf0258702051747439949737fc0b5
-        ) {
-            bytes memory _queryData = abi.encode(
-                "SpotPrice",
-                abi.encode("eth", "usd")
-            );
-            _id = keccak256(_queryData);
-            _isERC362Id = true;
-            _decimalsDividend = 1e15;
-        } else if (
-            _id ==
-            0x637b7efb6b620736c247aaa282f3898914c0bef6c12faff0d3fe9d4bea783020
-        ) {
-            bytes memory _queryData = abi.encode(
-                "SpotPrice",
-                abi.encode("btc", "usd")
-            );
-            _id = keccak256(_queryData);
-            _isERC362Id = true;
-            _decimalsDividend = 1e15;
-        } else if (
-            _id ==
-            0x2dfb033e1ae0529b328985942d27f2d5a62213f3a2d97ca8e27ad2864c5af942
-        ) {
-            bytes memory _queryData = abi.encode(
-                "SpotPrice",
-                abi.encode("xau", "usd")
-            );
-            _id = keccak256(_queryData);
-            _isERC362Id = true;
-            _decimalsDividend = 1e15;
-        } else if (
-            _id ==
-            0x9899e35601719f1348e09967349f72c7d04800f17c14992d6dcf2f17fac713ea
-        ) {
-            bytes memory _queryData = abi.encode(
-                "SpotPrice",
-                abi.encode("dai", "usd")
-            );
-            _id = keccak256(_queryData);
-            _isERC362Id = true;
-            _decimalsDividend = 1e12;
-        }
-
+        _id = idMappingContract.getTellorID(_id);
```

Comment:
should the mapping contract also return the number of decimals since our SpotPrice uses 18 and [this EIP2362 doc](https://github.com/adoracles/EIPs/blob/erc-2362/EIPS/eip-2362.md#registered-ids) shows different decimal places?